strings/case_style.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+export enum caseStyle {
+  pascalCase = "pascalCase",
```

Comment:
Should we make an example for every `caseStyle`?

Developers know more intuitively which style they need

```suggestion
  pascalCase = "pascalCase", // eg. `LastName`
```